src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      calculator.result += num;
+
+      return calculator;
+    },
+    subtract(num) {
+      calculator.result -= num;
+
+      return calculator;
+    },
+    multiply(num) {
+      calculator.result *= num;
+
+      return calculator;
+    },
+    divide(num) {
+      calculator.result /= num;
```

Comment:
Replace 'calculator.result' with 'this.result' to access the current object's property.